content/articles/url-record.markdown

```diff
@@ -24,3 +24,36 @@ Enter the hostname you would like to redirect from, and the URL to redirect to.
 ## Redirecting over SSL
 
 Right now it is not possible for our redirection service to handle SSL requests for your domain names. Take a look at the article [Can a URL record redirect requests over SSL?](/articles/url-redirect-ssl) for additional information and an explanation of alternatives.
+
+## URL to URL redirects
+
+URL to URL redirects can't be done with our URL record. Only domains or subdomains can be redirected to complete URLs. The following redirect will not work:
+
+    foo.com/blog to bar.com/blog
```

Comment:
I don't think this example fully explains what we're trying to say here. What about this:

> The following redirect will not work:
>
>     foo.com/blog/ to bar.com/blog
>
> because DNS records cannot be added to URLs. However, this would work:
>
>     blog.foo.com to bar.com/blog
>
> since a URL record could be added to blog.foo.com.